modules/districtmDMXBidAdapter.js

```diff
@@ -27,9 +39,14 @@ export const spec = {
               nBid.requestId = nBid.impid;
               nBid.width = nBid.w || width;
               nBid.height = nBid.h || height;
+              nBid.mediaType = bid.mediaTypes && bid.mediaTypes.video ? 'video' : null;
+              if (nBid.mediaType) {
+                nBid.vastXml = cleanVast(nBid.adm);
+              }
               if (nBid.dealid) {
                 nBid.dealId = nBid.dealid;
               }
+              nBid.uuid = nBid.bidId;
               nBid.ad = nBid.adm;
               nBid.netRevenue = true;
```

Comment:
can you also write nBid.adomain to meta.advertiserDomains as per #3115 ? You can see a bunch of example commits doing this at the bottom of the issue. I just hit the dmx endpoinnt and saw adomain and cat populated as arrays.

Reply:
this was update